File: front/src/data/TeamStats.ts
import { Team } from "./Team";

export interface TeamStats {
    team: Team;
    played: Number;
    won: Number;
    drawn: Number;
    lost: Number;
    score: Number;
    pointsFor: Number;
    pointsAgainst: Number;
    pointsDiff: Number;
}

File: api/src/main/java/com/gberard/tournament/domain/model/stats/TeamStats.java
package com.gberard.tournament.domain.model.stats;

import com.gberard.tournament.domain.model.Team;

public record TeamStats(
        Team team,
        int played,
        int won,
        int drawn,
        int lost,
        int score,
        int pointsFor,
        int pointsAgainst,
        int pointsDiff
) {
}
